gromacs/config.py

```diff
@@ -222,14 +201,6 @@
 
    gromacs.grompp(f="md.mdp", c="system.gro", p="topol.top", o="md.tpr")
 
-(The driver command is stripped and only the "command name" is used to
```

Comment:
The new `configuration.txt` file should also say under which names tools appear. Eg if a driver is `gmx` then it will generate `gromacs.mdrun` from `gmx mdrun` but if the driver is `gmx_d` the this will be `gromacs.mdrun_d` (I think... *I* would like to see the docs myself).